assets/blocks/view-certificate-block/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { registerBlockVariation } from '@wordpress/blocks';
+
+const buttonVariationAttributes = {
+	url: '/course/sensei-view-certificate',
+	title: __( 'View Certificate', 'sensei-certificates' ),
+	text: __( 'View Certificate', 'sensei-certificates' ),
+	linkTarget: '_blank'
+};
+
+registerBlockVariation( 'core/button', {
+	name: 'view-certificate-button',
+	attributes: buttonVariationAttributes
```

Comment:
I noticed that the Button block title is the same when trying to add a new button to the Buttons block. I also tried to add a title in this block, but it didn't update in the InnerBlocks. I think probably it's a Gutenberg issue - In the sidebar, and in the List View, it also has the original block titles.

<img width="639" alt="Screen Shot 2021-04-07 at 11 34 48" src="https://user-images.githubusercontent.com/876340/113884358-48121d00-9795-11eb-98b7-6526f23063e9.png">

Reply:
Sorry my bad. It seems that I broke this in a last minute refactor. Fixed in cb0aa85bca5dbf5c231cddcbe16ab6cd7b0c84ad

Although this appears with a different name and description in the inserter, once the block is inserted then it will have the default button values. This kind of makes sense since a variation might be a combination of many different blocks.